src/PrintLoopNest.cpp

```diff
@@ -197,5 +198,19 @@ string print_loop_nest(const vector<Function> &output_funcs) {
     return sstr.str();
 }
 
+string print_loop_nest(const vector<Function> &output_funcs) {
+    // For the purposes of printing the loop nest, we don't want to
+    // worry about which features are and aren't enabled.
+    Target target = get_host_target();
+    for (DeviceAPI api : all_device_apis) {
+        target.set_feature(target_feature_for_device_api(DeviceAPI(api)));
+    }
+    // target_feature_for_device_api doesn't handle DeviceAPI::Hexagon becuase
```

Comment:
I chose to do it this way so as to localize the hack rather than expose it to the rest of the source base by putting it inside target_feature_for_device_api (although print_loop_nest is the only user of this function). But, I'll put this in target_feature_for_device_api if you still prefer so.